metric_learn/base_metric.py

```diff
@@ -85,6 +94,24 @@ def _prepare_inputs(self, X, y=None, type_of_inputs='classic',
                        tuple_size=getattr(self, '_tuple_size', None),
                        **kwargs)
 
+  @abstractmethod
+  def get_metric(self):
+    """Returns a function that returns the learned metric between two points.
```

Comment:
Returns a function that takes as input two 1D arrays and outputs the learned metric score on these two points?

Reply:
Agreed it's clearer indeed